dali/operators/reader/webdataset_reader_op.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_READER_WEBDATASET_READER_OP_H_
+#define DALI_OPERATORS_READER_WEBDATASET_READER_OP_H_
+
+#include "dali/operators/reader/loader/webdataset_loader.h"
+#include "dali/operators/reader/reader_op.h"
+#include "dali/pipeline/data/tensor.h"
+
+namespace dali {
+
+class WebdatasetReader : public DataReader<CPUBackend, vector<Tensor<CPUBackend>>> {
+ public:
+  explicit WebdatasetReader(const OpSpec& spec)
+      : DataReader<CPUBackend, vector<Tensor<CPUBackend>>>(spec) {
+    std::cout << "TEST" << std::endl;
+    std::cout << "TEST" << std::endl;
+    std::cout << "TEST" << std::endl;
+    std::cout << "TEST" << std::endl;
```

Comment:
?

Reply:
some tests, stuff, that's why it's still WIP